python/tests/mlir/test_output_translate_openqasm.py

```diff
@@ -0,0 +1,40 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+# RUN: PYTHONPATH=../../ pytest -rP  %s | FileCheck %s
+
+import os
+
```

Comment:
Why not stick all the import statements in a block without whitespace? Maybe the formatting tool is doing something here?

FWIW: With C++ files, the `#include` directives shouldn't have random blank lines stuck here and there.

Reply:
Done!